### ISSUE TITLE
各位大佬，视觉感知算法集成开发示例 训练模型出错，提示  nvcc - Not Available(小白学习)

### ISSUE THREAD

**Issue by TreasureYi** [PaddlePaddle/Paddle3D#220]
LAUNCH INFO 2023-01-17 20:42:49,572 ------------------------- ERROR LOG DETAIL -------------------------
3.5
    numba - 0.56.4
    pandas - 1.5.2
    pillow - 8.3.2
    skimage - 0.19.3

PaddlePaddle:
    paddle(gpu) - 2.4.1
    paddle3d - 1.0.0
    paddleseg - 2.7.0
    FLAGS_cudnn_deterministic - Not set.
    FLAGS_cudnn_exhaustive_search - Not set.

CUDA:
    cudnn - 7605
    nvcc - Not Available

GPUs:
------------------------------------------------
2023-01-17 20:42:29,241 -     INFO -
---------------Config Information---------------
batch_size: 8
export:
  transforms:
  - reader: pillow
    to_chw: false
    to_rgb: true
    type: LoadImage
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
iters: 100
lr_scheduler:
  learning_rate: 0.000125
  milestones:
  - 5000
  - 8000
  type: MultiStepDecay
model:
  backbone:
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/hrnet_w18_ssld.tar.gz
    type: $paddleseg.HRNet_W18
  depth_ref:
  - 28.01
  - 16.32
  dim_ref:
  - - 3.88
    - 1.63
    - 1.53
  - - 1.78
    - 1.7
    - 0.58
  - - 0.88
    - 1.73
    - 0.67
  head:
    in_channels: 270
    norm_type: gn
    num_chanels: 256
    num_classes: 3
    reg_channels:
    - 1
    - 2
    - 3
    - 2
    - 2
    type: SMOKEPredictor
  max_detection: 50
  pred_2d: true
  type: SMOKE
optimizer:
  type: Adam
train_dataset:
  dataset_root: datasets/KITTI
  mode: train
  transforms:
  - reader: pillow
    to_chw: false
    type: LoadImage
  - input_size:
    - 1280
    - 384
    mode: train
    num_classes: 3
    type: Gt2SmokeTarget
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
  type: KittiMonoDataset
val_dataset:
  dataset_root: datasets/KITTI
  mode: val
  transforms:
  - reader: pillow
    to_chw: false
    type: LoadImage
  - input_size:
    - 1280
    - 384
    mode: val
    num_classes: 3
    type: Gt2SmokeTarget
  - mean:
    - 0.485
    - 0.456
    - 0.406
    std:
    - 0.229
    - 0.224
    - 0.225
    type: Normalize
  type: KittiMonoDataset
------------------------------------------------
W0117 20:42:29.243573 17772 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 10.2, Ru
ntime API Version: 10.2
W0117 20:42:29.243573 17772 gpu_resources.cc:91] device: 0, cuDNN Version: 7.6.
2023-01-17 20:42:31 [INFO]      Loading pretrained model from https://bj.bcebos.com/paddleseg/dygraph/hrnet_w18_ssld.tar.gz
2023-01-17 20:42:32 [INFO]      There are 1525/1525 variables loaded into HRNet.
E:\develop\miniconda3\conda-data\conda-env\paddle_env\lib\site-packages\paddle\nn\layer\norm.py:711: UserWarning: When training, w
e now always track global mean and variance.
  warnings.warn(
E:\develop\miniconda3\conda-data\conda-env\paddle_env\lib\site-packages\paddle\nn\layer\norm.py:711: UserWarning: When training, w
e now always track global mean and variance.
  warnings.warn(
LAUNCH INFO 2023-01-17 20:42:49,625 Exit code 3221226505

**Comment by yijianpeng** [comment]
请问你是怎么解决的，我nvcc -V能输出CUDA11.6
